ocpmodels/trainers/base_trainer.py

```diff
@@ -505,7 +511,8 @@ def _compute_loss(self, out, batch):
             else:
                 grad_target_normed = batch.force
             loss.append(
-                self.config["optim"].get("force_coefficient", 30) * self.criterion(out["force_output"], grad_target_normed)
+                self.config["optim"].get("force_coefficient", 30)
```

Comment:
what's the intuition behind the value 30?

Reply:
Just something that worked well for us in practice; I can add a comment there.